src/components/OutsideNavbar.tsx

```
@@ -197,15 +227,30 @@ const OutsideNavbar: React.FC = () => {
+                          alignItems: 'center',
+                          px: 2
+                        }}
+                      >
+                        {item.label}
+                      </Button>
+                    ) : (
+                      <SubmenuDropdown 
+                        key={item.label} 
+                        label={item.label} 
+                        path={item.path}
+                        children={item.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)